spotty/spotty/Scenes/Core/MusicScene/MusicViewModel.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  MusicViewModel.swift
+//  spotty
+//
+//  Created by Ivaylo Kalaydzhiev on 16.06.22.
+//
+
+import UIKit
+
+// TODO: Create protocol
+class MusicViewModel {
+    
+    let featuredPlaylists = Observable([Playlist]())
+    let recentlyPlayedTracks = Observable([AudioTrack]())
+    let recentlyPlayedArtists = Observable([Artist]())
+    
+    private let webRepository: WebRepository
+    
+    init(webRepository: WebRepository = WebRepository()) {
+        self.webRepository = webRepository
+        
+        webRepository.getRecentlyPlayedTracks { [weak self] result in
```

Comment:
As we discussed we want to extract this in _start_ function and call in in the view controller. Refactor in this PR or mark with TODO

Reply:
I made a TODO